# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/ffmpeg/compile.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN ffmpeg-0.4.9-pre1/Makefile ffmpeg-0.4.9-pre1-new/Makefile
--- ffmpeg-0.4.9-pre1/Makefile	2004-07-08 13:02:04.000000000 +0200
+++ ffmpeg-0.4.9-pre1-new/Makefile	2004-07-26 12:55:08.261050798 +0200
@@ -14,6 +14,9 @@
 LDFLAGS+=-p
 endif
 
+# ReneR: well the maintainer could also test his ... without imlib2 support
+EXTRALIBS+=-L/usr/X11/lib -lX11
+
 MANPAGE=$(SRC_PATH)/doc/ffmpeg.1
 PROG=ffmpeg$(EXESUF)
 PROGTEST=output_example$(EXESUF)
